hieradata/site/ls/role/bastion.yaml

```diff
@@ -1,10 +1,13 @@
 ---
 classes:
   - "profile::core::nfsclient"
+  - "profile::core::icinga_agent"
 
 nfs::client_enabled: true
 nfs::client_mounts:
   /net/lsstdata/admin:
     share: "lsstdata/admin"
     server: "ddn-nfs.ls.lsst.org"
     atboot: true
+
+profile::core::icinga_agent::host_template: "GeneralHostTemplate"
```

Comment:
As this seems to be the general default, this key should go into `org/lsst.yaml` and be de-duplicated.

Reply:
done as suggested

Reply:
It is great that the key is now in `hieradata/org/lsst.yaml` but it is duplicated here unnecessarily.  The literal line `profile::core::icinga_agent::host_template: "GeneralHostTemplate"` should be removed from all roles as this is now duplicating the default.